test/tools/ossfuzz/AbiV2IsabelleFuzzer.cpp

```diff
@@ -58,8 +58,9 @@ DEFINE_PROTO_FUZZER(Contract const& _contract)
 		// We target the default EVM which is the latest
 		langutil::EVMVersion version;
 		EVMHost hostContext(version, evmone);
-		string contractName = ":C";
```

Comment:
I think you should update the docstring for `CompilerInput.contractName` to say how the value is supposed to look like. Especially whether it requires the colon or not.

Reply:
Updated.